webui/react/src/Analytics.ts

```diff
@@ -48,21 +48,27 @@ const getReadyAnalytics = (): SegmentAnalytics.AnalyticsJS | undefined => {
   return undefined;
 };
 
-export const setupAnalytics = async (info: DeterminedInfo): Promise<void> => {
+export const setupAnalytics = async (auth: Auth, info: DeterminedInfo): Promise<void> => {
   if (!data.analytics) data.analytics = getReadyAnalytics();
   if (!data.analytics || data.isEnabled) return;
 
   /*
    * Segment key should be 32 characters composed of upper case letters,
    * lower case letters and numbers 0-9.
    */
-  if (info.isTelemetryEnabled) {
+  if (auth.user && info.isTelemetryEnabled) {
```

Comment:
question: do we want to collect visits to unauthenticated pages? so far only the login page.

Reply:
separated out the identify from load so we can still get analytics going for unauthenticated flow